### PR TITLE
Coupe and Sedan keyboards

### PR DESCRIPTION
**Pull request by AlisGraveNil**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [X] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [X] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/clawsome/coupe/info.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
+{
+    "keyboard_name": "Coupe",
+    "url": "www.clawboards.xyz",
+    "maintainer": "AAClawson (AlisGraveNil)",
+    "width": 15,
+    "height": 5,
+    "layouts": {
+        "LAYOUT": {
+            "key_count": 61,
```
Comment (fauxpark) [inline keyboards/clawsome/coupe/info.json:9]:
```suggestion
```

File: keyboards/clawsome/coupe/readme.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#Coupe
```
Comment (fauxpark) [inline keyboards/clawsome/coupe/readme.md:1]:
```suggestion
# Coupe
```

File: keyboards/clawsome/sedan/info.json [hunk 3]
```diff
@@ -0,0 +1,84 @@
+{
+    "keyboard_name": "sedan",
+    "url": "www.clawboards.xyz",
+    "maintainer": "AAClawson (AlisGraveNil)",
+    "width": 16,
+    "height": 5,
+    "layouts": {
+        "LAYOUT": {
+            "key_count": 68,
```
Comment (fauxpark) [inline keyboards/clawsome/sedan/info.json:9]:
```suggestion
```

File: keyboards/clawsome/sedan/readme.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+#Sedan
```
Comment (fauxpark) [inline keyboards/clawsome/sedan/readme.md:1]:
```suggestion
# Sedan
```

File: keyboards/clawsome/sedan/readme.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+This is a 5x15 keyboard in a 68% layout.
+
+* Keyboard Maintainer: [AAClawson](http://github.com/AlisGraveNil)
+* Hardware Supported: The Sedan, Pro Micro, Elite-C
+* Hardware Availability: www.clawboards.xyz
+
+Make example for this keyboard (after setting up your build environment):
+
+	make clawsome/sedan:default
```
Comment (fauxpark) [inline keyboards/clawsome/sedan/readme.md:11]:
```suggestion
    make clawsome/sedan:default
```

File: keyboards/clawsome/sedan/config.h [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0000
```
Comment (fauxpark) [inline keyboards/clawsome/sedan/config.h:24]:
```suggestion
#define PRODUCT_ID      0x8C78
```
```
echo -n "The Sedan" | shasum | cut -c1-4
```

File: keyboards/clawsome/coupe/readme.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+
+This is a 5x14 keyboard in a 60% layout.
+
+* Keyboard Maintainer: [AAClawson](http://github.com/AlisGraveNil)
+* Hardware Supported: The Coupe, Pro Micro, Elite-C
+* Hardware Availability: www.clawboards.xyz
+
+Make example for this keyboard (after setting up your build environment):
+
+	make clawsome/coupe:default
```
Comment (fauxpark) [inline keyboards/clawsome/coupe/readme.md:11]:
```suggestion
    make clawsome/coupe:default
```

File: keyboards/clawsome/sedan/keymap/default/keymap.c [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
+        
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,          KC_HOME,
+        KC_TAB,      KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,      KC_PGUP,
+        KC_CAPS,       KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,              KC_PGDN,
+        KC_LSFT,            KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_LSFT,        XXXXXXX, KC_END,
+        KC_LCTL, KC_LGUI, KC_LALT, KC_SPC,  KC_LALT, XXXXXXX, KC_LCTL, XXXXXXX, XXXXXXX, XXXXXXX,     ),
```
Comment (noroadsleft) [inline keyboards/clawsome/sedan/keymap/default/keymap.c:25]:
```suggestion
        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,          KC_HOME,
        KC_TAB,      KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,      KC_PGUP,
        KC_CAPS,       KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,              KC_PGDN,
        KC_LSFT,            KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_LSFT,        XXXXXXX, KC_END,
        KC_LCTL, KC_LGUI, KC_LALT,                           KC_SPC,                              KC_LALT, XXXXXXX, KC_LCTL, XXXXXXX, XXXXXXX, XXXXXXX
    )
```

File: keyboards/clawsome/sedan/rules.mk [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no             # USB Nkey Rollover
+BACKLIGHT_ENABLE = no        # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no         # Enable keyboard RGB underglow
+MIDI_ENABLE = no             # MIDI support
+UNICODE_ENABLE = no          # Unicode
+BLUETOOTH_ENABLE = no        # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no            # Audio output on port C6
+FAUXCLICKY_ENABLE = no       # Use buzzer to emulate clicky switches
+
```
Comment (noroadsleft) [inline keyboards/clawsome/sedan/rules.mk:26]:
```suggestion

LAYOUTS = 65_ansi
```

File: keyboards/clawsome/sedan/sedan.h [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT( \
```
Comment (noroadsleft) [inline keyboards/clawsome/sedan/sedan.h:20]:
```suggestion
#define LAYOUT_65_ansi( \
```

File: keyboards/clawsome/sedan/sedan.h [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+    K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D,      K0E,   \
+    K10,   K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D,    K1E,   \
+    K20,    K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, K2D,        K2E,   \
+    K30,       K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B,     K3D, K3E,   \
+    K40,            K41, K42, K45, K49, K4A, K4B, K4C,                    K4D, K4E,   \
```
Comment (noroadsleft) [inline keyboards/clawsome/sedan/sedan.h:25]:
```suggestion
    K40,            K41, K42, K45, K49, K4A, K4B, K4C,                    K4D, K4E    \
```

**Comment by AlisGraveNil** [comment]
@noroadsleft - I used the KBFirmware-to-QMK site, then tried to match the new files with the approved files from the macropads I've had approved previously, in an attempt at reducing the workload/review you and the other admins do. Some of that may have backfired if you're seeing bugs now.

**Comment by noroadsleft** [comment]
> @noroadsleft - I used the KBFirmware-to-QMK site, then tried to match the new files with the approved files from the macropads I've had approved previously, in an attempt at reducing the workload/review you and the other admins do. Some of that may have backfired if you're seeing bugs now.

If you could DM me the Sedan's KBFirmware JSON on Discord, I'd appreciate it.

**Comment by AlisGraveNil** [comment]
Well I feel like a dingus

**Comment by AlisGraveNil** [comment]
welp...I tried to build out the keymaps as suggested, and apparently now it thinks I changed 86 other files...

**Comment by noroadsleft** [comment]
> welp...I tried to build out the keymaps as suggested, and apparently now it thinks I changed 186 other files...

Sent you a pull request with two commits. One fixes the extra files added, and the other extends the keymap functionality.

Merge it and it will fix this PR.

**Comment by AlisGraveNil** [comment]
It didn't even occur to me that you were referring to a second layer. I have macropads on my desktop for the F-keys and arrow keys and I didn't even think about that for other users.

Apologies if I seemed incredibly obtuse...

File: keyboards/clawsome/coupe/readme.md [hunk 12]
```diff
@@ -0,0 +1,13 @@
+# Coupe
+
```
Comment (Erovia) [inline keyboards/clawsome/coupe/readme.md:2]: I like having at least a link to a picture about the board.  
Can help tremendously while helping others. (so preferably from a source that will be available long-term)

File: keyboards/clawsome/sedan/readme.md [hunk 13]
```diff
@@ -0,0 +1,13 @@
+# Sedan
+
```
Comment (Erovia) [inline keyboards/clawsome/sedan/readme.md:2]: Same thing about pictures.

File: keyboards/clawsome/coupe/config.h [hunk 14]
```diff
@@ -0,0 +1,47 @@
...
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
```
Comment (Erovia) [inline keyboards/clawsome/coupe/config.h:23]: It's only a matter of time until someone decides to create a via keymap for these boards.  
At that point they will find a unique VID for you.  
If you'd rather choose one for yourself, now is the time. :)

File: keyboards/clawsome/sedan/config.h [hunk 15]
```diff
@@ -0,0 +1,47 @@
...
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
```
Comment (Erovia) [inline keyboards/clawsome/sedan/config.h:23]: Same thing about VID.

File: keyboards/clawsome/coupe/config.h (hunk 14 above)
Comment (AlisGraveNil) [reply keyboards/clawsome/coupe/config.h:23]: I love VIA, but I have no idea how to choose a VID for myself

File: keyboards/clawsome/coupe/readme.md (hunk 12 above)
Comment (AlisGraveNil) [reply keyboards/clawsome/coupe/readme.md:2]: I can link to the shop page from my website - should I add the URL there to line 2?

File: keyboards/clawsome/coupe/config.h (hunk 14 above)
Comment (fauxpark) [reply keyboards/clawsome/coupe/config.h:23]: The usual convention is to use two ASCII characters, eg. the VID for the 1upkeyboards boards is `0x6F75` for `ou`. You can check https://yanfali.github.io/qmk_usb_usage/ to see which values are taken.

File: keyboards/clawsome/coupe/readme.md (hunk 12 above)
Comment (Erovia) [reply keyboards/clawsome/coupe/readme.md:2]: Add a line like this:

`Coupe`

File: keyboards/clawsome/sedan/config.h (hunk 15 above)
Comment (AlisGraveNil) [reply keyboards/clawsome/sedan/config.h]: added VID (0x7767)

File: keyboards/clawsome/coupe/config.h (hunk 14 above)
Comment (AlisGraveNil) [reply keyboards/clawsome/coupe/config.h]: added VID (0x7767)

File: keyboards/clawsome/coupe/readme.md (hunk 12 above)
Comment (AlisGraveNil) [reply keyboards/clawsome/coupe/readme.md:2]: Added to line #3

File: keyboards/clawsome/sedan/readme.md (hunk 13 above)
Comment (AlisGraveNil) [reply keyboards/clawsome/sedan/readme.md:2]: added to line 3

File: keyboards/clawsome/coupe/readme.md [hunk 16]
```diff
@@ -1,5 +1,7 @@
 # Coupe
 
+![Coupe](https://www.clawboards.xyz/shop/coupe)
```
Comment (noroadsleft) [inline keyboards/clawsome/coupe/readme.md:3]: The goal here is to have a direct photo of the keyboard here. (I really wish Squarespace URLs weren't so hideous.)

```suggestion
![Coupe](https://images.squarespace-cdn.com/content/v1/5ec961924eb3960db955f752/1591981988950-D64UDSNKPAT91WUA8RNC/ke17ZwdGBToddI8pDm48kDHPSfPanjkWqhH6pl6g5ph7gQa3H78H3Y0txjaiv_0fDoOvxcdMmMKkDsyUqMSsMWxHk725yiiHCCLfrh8O1z4YTzHvnKhyp6Da-NYroOW3ZGjoBKy3azqku80C789l0mwONMR1ELp49Lyc52iWr5dNb1QJw9casjKdtTg1_-y4jz4ptJBmI9gQmbjSQnNGng/Coupe3.jpeg?format=640w)
```

File: keyboards/clawsome/sedan/readme.md [hunk 17]
```diff
@@ -1,5 +1,7 @@
 # Sedan
 
+![Sedan](https://www.clawboards.xyz/shop/sedan)
```
Comment (noroadsleft) [inline keyboards/clawsome/sedan/readme.md:3]:
```suggestion
![Sedan](https://images.squarespace-cdn.com/content/v1/5ec961924eb3960db955f752/1592952281584-LTU6E70P5WIFWPZR56SH/ke17ZwdGBToddI8pDm48kDHPSfPanjkWqhH6pl6g5ph7gQa3H78H3Y0txjaiv_0fDoOvxcdMmMKkDsyUqMSsMWxHk725yiiHCCLfrh8O1z4YTzHvnKhyp6Da-NYroOW3ZGjoBKy3azqku80C789l0mwONMR1ELp49Lyc52iWr5dNb1QJw9casjKdtTg1_-y4jz4ptJBmI9gQmbjSQnNGng/Sedan1.jpeg?format=640w)
```

File: keyboards/clawsome/coupe/readme.md (hunk 16 above)
Comment (Erovia) [reply keyboards/clawsome/coupe/readme.md:3]: Maybe a "mirror" on imgur would be best.

**Comment by noroadsleft** [comment]
Thanks!

File: keyboards/clawsome/sedan/config.h [hunk 18]
```diff
@@ -0,0 +1,47 @@
...
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+ */
+#define MATRIX_ROW_PINS { B0, B1, B2, B3, B4 }
+#define MATRIX_COL_PINS { B5, B6, B7, C0, C1, C2, C3, C4, C5, C6, C7, D0, D1, D2, D3 }
```
Comment (beansrowning) [inline keyboards/clawsome/sedan/config.h:45]: @AlisGraveNil Is this accurate to an elite-C? I'm debugging a fully built board that has an issue with multiple key-downs and I've never seen a pinout diagram of an elite C look like this (no F pins, C0?). 

Can open a separate issue if this is truly off, but I'm trying to figure out if I'm just going crazy or not.

Comment (beansrowning) [reply keyboards/clawsome/sedan/config.h:45]: To clarify, based on The Sedan v1.2 that's available

Comment (AlisGraveNil) [reply keyboards/clawsome/sedan/config.h:45]: I have no idea where it's getting 'C0' from - that's not at all the pinout I created last year.

Do you have a Sedan on hand that's bugging out?

Comment (beansrowning) [reply keyboards/clawsome/sedan/config.h:45]: I do! Happy to assist.

Comment (AlisGraveNil) [reply keyboards/clawsome/sedan/config.h:45]: email me (alex@clawboards.xyx) and I'll send you a working firmware that I've had archived, and I'll look at doing a fork for fixes in a bit

**Pull request merged**